recipe_scrapers/projectgezond.py

```diff
@@ -10,83 +8,3 @@ def host(cls):
 
     def author(self):
         return "Project Gezond"
-
-    def title(self):
-        return self.soup.find("h1", {"class": "entry-title"}).text
-
-    def category(self):
```

Comment:
It doesn't appear super-straightforward to retrieve, but I notice that `category` data is still available on the page, for example in the `dataLayer_content` JS variable.

Reply:
just pushed an update with this!